src/net/sourceforge/kolmafia/textui/parsetree/Command.java

```diff
@@ -0,0 +1,40 @@
+package net.sourceforge.kolmafia.textui.parsetree;
+
+public abstract class Command
+	extends ParseTreeNode
+{
+	// A barrier is any code construct that is fundamentally incapable of
```

Comment:
Convert this to a multiline comment, and add blank lines between paragraphs? Otherwise this is just a wall of text that I don't feel like reading.

(ditto below.)

(Although, reading on, I see this was lifted directly from ParseTreeNode...)